mail/rspamd/Portfile

```diff
@@ -33,49 +30,80 @@ compiler.whitelist clang macports-clang-8.0 macports-clang-7.0 macports-clang-6.
 
 depends_build-append \
                     port:pkgconfig \
+                    port:ragel \
                     port:luajit \
-                    port:ragel
+                    port:gd2 \
+                    port:fann \
+                    port:libstemmer
```

Comment:
Just double checking, these 3 additions are really only build deps ? They aren’t needed at runtime ?

Reply:
Please don’t resolve a comment with actually saying or changing anything. If its ok then fine, but a comment to that affect would be good.

Reply:
Thanks, corrected. See https://github.com/macports/macports-ports/pull/4258/files#r280977351

Reply:
Hold on. I see that there's a `libragel.dylib` too. `port:ragel` needs to go under `depends_lib`.

Also note that I added a `depends_run` section for ports wanted at runtime.